Userland/Libraries/LibWeb/Layout/TableFormattingContext.cpp

```diff
@@ -1110,27 +1110,15 @@ void TableFormattingContext::position_cell_boxes()
                 cell_state.padding_top += row_content_height - cell_state.border_box_height();
                 break;
             }
+            case CSS::VerticalAlign::Sub:
```

Comment:
Please add a layout test so people can test locally without needing to look at your PR :-)